app/controllers/clients/files_controller.rb

```diff
@@ -30,6 +32,10 @@ def create
       end
       redirect_to action: :index 
     end
+
+    def update
+      @file.update(file_params)
```

Comment:
This will need to verify that the user can actually set the `:consent_form_confirmed` flag before allowing it through.